server/src/main/java/com/broadcom/lsp/domain/cobol/databus/impl/DefaultDataBusBroker.java

```diff
@@ -92,31 +90,21 @@ public void unSubscribe(@NonNull RegistryId registryId, @NonNull S dataSubscribe
   }
 
   @Override
-  public CopybookStorable storeData(@NonNull CopybookStorable storable) {
-    CopybookStorable deepCopy = SerializationUtils.clone(storable);
```

Comment:
Shouldn't we copy the object? Or it is assured by the `Cache`?

Reply:
We don't need to copy an object in the current implementation. The thing is that `CopybookStorable` is modifiable, and maybe that why the object is copied before storing. But current implementation uses `CopybookModel` which is unmodifiable, it is safe to use one instance everywhere.